lib/msal-core/src/ErrorMessage.ts

```diff
@@ -21,13 +21,127 @@
   * OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
   */
 
-/**
- * @hidden
- */
-export class ErrorMessage {
-  static get authorityUriInvalidPath(): string { return "AuthorityUriInvalidPath"; }
-  static get authorityUriInsecure(): string { return "AuthorityUriInsecure"; }
-  static get invalidAuthorityType(): string { return "InvalidAuthorityType"; }
-  static get unsupportedAuthorityValidation(): string { return "UnsupportedAuthorityValidation"; }
-  static get b2cAuthorityUriInvalidPath(): string { return "B2cAuthorityUriInvalidPath"; }
-}
+export const ErrorMessage = {
```

Comment:
If we break up every error class, i actually like the idea better of putting these config in the individual files 

so `ClientInteractionError.ts` would have the errors for it, and its still defined in a const at the top of the file, but its also really easy to see the definition

Reply:
I think we wanted to consolidate strings to a single file, bu this could work too. I'd like the rest of the team's feedback as well.

Reply:
Yeah i think having one file has its benefits but I really like the idea of small digestable files 

FakeErrorClass.ts
```js
import { AuthError } from './AuthError';

const  fakeErrors = {
   errorone: {
          code: 'error_one',
          message: 'I want to update error one and its here with the error class!'
   },
  errortwo:  {
          code: 'error_one',
          message: 'And since there are only two errors here its super easy to read and digest these errors'
   },
};

export default class FakeErrorClass extends AuthError { 
   constructor(cool) {
      super(cool)
   }

   static createErrorOne(message: string) {
            ....
   } 

   static createErrorTwo(message: string) {
            ....
   } 
}
```